readme.md

```diff
@@ -65,7 +82,7 @@ Describe the purpose of views and models in FE JS.
 
 Your Answer:
 ```text
-
+The models contain the constructor functions that will be used to create new client-side instances of an object. Views are then used to render objects on the client-side.
 ```
```

Comment:
:+1:

Views also let us add functionality to a web app (e.g., via event listeners).